app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -191,23 +158,16 @@ public void onClick(DialogInterface dialog, int which) {
     public void startRecording() {
         Context ctx = getApplicationContext();
         if (requestLocationPermission()) {
-            LocationService.start(this);
+            Recorder.startService(this);
             Toast.makeText(ctx, "Location service started", Toast.LENGTH_SHORT).show();
             Log.v(App.TITLE, "Location service started");
         } else {
             Toast.makeText(ctx, "no permission -- try again", Toast.LENGTH_SHORT).show();
         }
     }
 
-    public void stopRecording() {
-        LocationService.stop(this);
-        Context ctx = getApplicationContext();
-        Toast.makeText(ctx, "Location service stopped", Toast.LENGTH_SHORT).show();
-        Log.v(App.TITLE, "Location service stopped");
-        exportButton.setEnabled(true);
-        clearButton.setEnabled(true);
-    }
 
+    // todo: Reconnect with a timer or listener, to update every x =D
```

Comment:
Should we add a ticket on the board for this? Is this the actual location logging or just pretty logging?